### DISCUSSION TITLE
vite 为基座  webpack3 子应用 会遇接口提示跨域问题

### DISCUSSION THREAD

**Discussion by you-hei-mi** [jd-opensource/micro-app#709] [Q&A]
子应用已配置
1、vue.config.js 开启跨域
devServer: {
    headers: {
      'Access-Control-Allow-Origin': '*'
    },
}
2、request.js 拦截器与接口请求都没有写死域名和端口
-------------------------------------
子应用页面也被正常渲染，但是接口提示跨域。
![image](https://user-images.githubusercontent.com/33774921/217233413-253c65c3-0f46-4ab3-b73a-3f915a2725ff.png)

报 Network Error 错误。

**Comment by soyeon-lq** [comment]
今天在试用micro是也遇到这个问题了，集成老项目请求时接口提示跨域，将下面代码配置到vue.config.js里面就可以了。
headers: {
      'Access-Control-Allow-Origin': '*',
      'Access-Control-Allow-Methods': '*',
      'Access-Control-Allow-Headers': '*'
    }
